### DISCUSSION TITLE
Template structure

### DISCUSSION THREAD

**Discussion by rogueturnip** [codex-team/editor.js#2278] [General]
Hi!  I'm hoping I might get some guidance on how to implement a template in editor.js

I have a structure for a "document" that I want to enforce with headers that can't be changed or edited and following text boxes that can be edited/filled in but not removed.

Is there any way in the data structure to indicate a block can't be removed and/or edited?

Thanks!

**Comment by neSpecc** [comment]
Currently there is no option for disable removing of a particular block. 

By the way, the templating is a nice-to-have feature.

**Reply by rogueturnip** [reply]
Thanks!  Is there any way to suggest such a feature to support templating?  It would really let EditorJs support use cases where free form content creation can't be allowed.

**Reply by neSpecc** [reply]
It should be designed first to make a decision about possibility of including it to the Roadmap.

Right now, our resources are overloaded. You can sponsor such a design session.

**Comment by ConnectGrid** [comment]
How about splitting up your "document" into several sections, each controlled by a separate instance of EditorJS? That would give you layout flexibility. The parts of the document that are static would not be an instance.
